client/src/templates/Challenges/utils/index.ts

```diff
@@ -3,7 +3,7 @@ import envData from '../../../../../config/env.json';
 const { forumLocation } = envData;
 
 interface GuideData {
-  forumTopicId: number;
+  forumTopicId?: number;
```

Comment:
Had to make `forumTopicId` property optional in order for test case `should use search endpoint when no forumTopicId is supplied` to pass, that's one thing that slipped out from us in last PR. Good thing that is covered with unit test :)

Reply:
Do you want us to resolve the conversation, looks like you intended to leave this note just as a comment?

Reply:
Sure, you can resolve it, I just wanted to leave a note